test_worker_node_restart_during_pvc_clone failed due to timeout while creating pod

Test case "tests/functional/pv/pvc_clone/test_node_restart_during_pvc_clone.py::TestNodeRestartDuringPvcClone::test_worker_node_restart_during_pvc_clone" failed while waiting for a pod to become Running.


```
# Verify the new pods are running
    log.info("Verify the new pods are running")
    for pod_obj in clone_pod_objs:

      wait_for_resource_state(pod_obj, constants.STATUS_RUNNING)

tests/functional/pv/pvc_clone/test_node_restart_during_pvc_clone.py:167: 



resource = <ocs_ci.ocs.resources.pod.Pod object at 0x7f1c1346ce50>
state = 'Running', timeout = 60


def wait_for_resource_state(resource, state, timeout=60):
    """
    Wait for a resource to get to a given status

    Args:
        resource (OCS obj): The resource object
        state (str): The status to wait for
        timeout (int): Time in seconds to wait

    Raises:
        ResourceWrongStatusException: In case the resource hasn't
            reached the desired state

    """
    if check_cluster_is_compact():
        timeout = 180
    if (
        resource.name == constants.DEFAULT_STORAGECLASS_CEPHFS
        or resource.name == constants.DEFAULT_STORAGECLASS_RBD
    ):
        logger.info("Attempt to default default Secret or StorageClass")
        return
    try:
        resource.ocp.wait_for_resource(
            condition=state, resource_name=resource.name, timeout=timeout
        )
    except TimeoutExpiredError:
        logger.error(f"{resource.kind} {resource.name} failed to reach {state}")
        resource.reload()

      raise ResourceWrongStatusException(resource.name, resource.describe())
E           ocs_ci.ocs.exceptions.ResourceWrongStatusException: Resource pod-test-cephfs-ca7a8a31685043a18d8a3721 describe output: Name:             pod-test-cephfs-ca7a8a31685043a18d8a3721
E           Namespace:        namespace-test-f43e2774edb24a17bd824a7fa
E           Priority:         0
E           Service Account:  default
E           Node:             compute-2/10.1.112.39
E           Start Time:       Tue, 05 Mar 2024 14:34:41 +0000
E           Labels:           
E           Annotations:      k8s.ovn.org/pod-networks:
E                               {"default":{"ip_addresses":["10.128.2.23/23"],"mac_address":"0a:58:0a:80:02:17","gateway_ips":["10.128.2.1"],"routes":[{"dest":"10.128.0.0...
E                             openshift.io/scc: anyuid
E           Status:           Pending
E           IP:
E           IPs:              
E           Containers:
E             web-server:
E               Container ID:
E               Image:          quay.io/ocsci/nginx:fio
E               Image ID:
E               Port:           
E               Host Port:      
E               State:          Waiting
E                 Reason:       ContainerCreating
E               Ready:          False
E               Restart Count:  0
E               Environment:    
E               Mounts:
E                 /var/lib/www/html from mypvc (rw)
E                 /var/run/secrets/kubernetes.io/serviceaccount from kube-api-access-cqsjl (ro)
E           Conditions:
E             Type              Status
E             Initialized       True
E             Ready             False
E             ContainersReady   False
E             PodScheduled      True
E           Volumes:
E             mypvc:
E               Type:       PersistentVolumeClaim (a reference to a PersistentVolumeClaim in the same namespace)
E               ClaimName:  clone-pvc-test-e238a68384524c-043095b072
E               ReadOnly:   false
E             kube-api-access-cqsjl:
E               Type:                    Projected (a volume that contains injected data from multiple sources)
E               TokenExpirationSeconds:  3607
E               ConfigMapName:           kube-root-ca.crt
E               ConfigMapOptional:       
E               DownwardAPI:             true
E               ConfigMapName:           openshift-service-ca.crt
E               ConfigMapOptional:       
E           QoS Class:                   BestEffort
E           Node-Selectors:              
E           Tolerations:                 node.kubernetes.io/not-ready:NoExecute op=Exists for 300s
E                                        node.kubernetes.io/unreachable:NoExecute op=Exists for 300s
E           Events:
E             Type    Reason     Age   From               Message
E             ----    ------     ----  ----               -------
E             Normal  Scheduled  77s   default-scheduler  Successfully assigned namespace-test-f43e2774edb24a17bd824a7fa/pod-test-cephfs-ca7a8a31685043a18d8a3721 to compute-2



ocs_ci/helpers/helpers.py:131: ResourceWrongStatusException
```


RUN ID is 1709628302
RP logs - https://reportportal-ocs4.apps.ocp-c1.prod.psi.redhat.com/ui/#ocs/launches/632/19122/928942/928966/log

--- a/tests/functional/pv/pvc_clone/test_node_restart_during_pvc_clone.py
+++ b/tests/functional/pv/pvc_clone/test_node_restart_during_pvc_clone.py
@@ -164,7 +164,7 @@ class TestNodeRestartDuringPvcClone(ManageTest):
         # Verify the new pods are running
         log.info("Verify the new pods are running")
         for pod_obj in clone_pod_objs:
-            wait_for_resource_state(pod_obj, constants.STATUS_RUNNING)
+            wait_for_resource_state(pod_obj, constants.STATUS_RUNNING, timeout=120)
         log.info("Verified: New pods are running")
 
         # Verify md5sum

